### PR TITLE
testing git

### PR DESCRIPTION
**Pull request by Mickeyjey**
just testing stuuf

### CODE REVIEW

File: expensewebsite/settings.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.2/ref/settings/
 """
+#decouple get values from .env
```
Comment (koonimo) [inline expensewebsite/settings.py:12]: nice

File: expensewebsite/settings.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
 from decouple import config
 from pathlib import Path
```
Comment (koonimo) [inline expensewebsite/settings.py:14]: all done

Comment (Mickeyjey) [reply expensewebsite/settings.py:14]: yes all done

**Review by koonimo — APPROVED** [review]
you can merge it now

File: expensewebsite/settings.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.2/ref/settings/
```
Comment (Mickeyjey) [inline expensewebsite/settings.py:10]: donw

**Pull request merged**